### DISCUSSION TITLE
Dengesiz para sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#9] [Zeka Sorusu]
Dengesiz bir para 125 kez atılıyor ve 35 kez tura geldiği gözlemleniyor. 500 kez daha atılırsa yazı gelme sayısının toplamda kaç olması beklenir?

**Comment by HabilCcabbar** [comment]
140?

**Reply by sonsuzus** [reply]
Soruları açıklamalı cevaplayalım. Doğru cevaptan önce düşünce biçimi önemli.

**Comment by adasue** [comment]
Öncelikle tura gelme olasılığının yüzde kaç olduğunu bulmamız gerekiyor o nedenle 35/125'i önce 7/25 sonra da 28/100 olarak yazıyoruz. Bize yazı gelme olasılığının 500'de toplamda kaç olduğunu bulmamızı istediği için 100/100'den 28/100'ü çıkarıyoruz. O da 72/100 oluyor. Sonrasında da 500'ü 72/100 ile çarpıyoruz ve **cevap 360** oluyor eğer tura gelme olasılığını sorsaydı da cevap 140 olurdu.

**Reply by sonsuzus** [reply]
Bir şeyi atlıyorsunuz ama. Para 500 defa atılmıyor. 500 kez daha atılıyor.

**Reply by adasue** [reply]
O zaman ilk başta 125 kere de atıldığı için toplamda 625 defa atılmış oluyor. Önceden de %72 ihtimalle yazı geldiğini bulmuştuk. Soruda da toplamda yazı gelme ihtimalini soruyor. 625'i 72/100 ile çarparsak **cevap 450** oluyor.

**Comment by Onuralp000** [comment]
İlk 125 atışta 35 tura geldiğine göre 125 atışın geri kalanı da yazı gelmiş olacaktır çünkü 125 toplam atış yapılıyor veya tura ya da yazı gelebilir.
Bu nedenle ilk 125 atışta 35 tura geldiyse, 125 atışın geri kalanında 125 - 35 = 90 yazı gelmiş olacak
Şimdi 500 kez daha para atıldığında, bu atışların sonucu ilk 125 atışın sonucunu değiştirmeyecektir. Yani bu 500 atışta da 90 yazı ve 410 tura gelmesi beklenir
Toplamda, 125 + 500 atışın sonucunda beklenen yazı gelme sayısı 90 + 90 = 180 olacaktır
